src/components/SignUp.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+
+const SignUp = () => {
```

Comment:
Use react hooks to save the form data and a http request to send it to backend, make the link as a environment variable for now, refer to this while creating env variables https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
the backend will be available on localhost:4000/api/v1
the user registration controller available on /api/v1/auth
make a POST request to the above route, sending the username, email, and the password as the form data.